Jenkinsfile.cloud

```diff
@@ -137,6 +137,13 @@ node(NODE) {
         }
     }
 
+   if (params.DRY_RUN) {
```

Comment:
I am still catching up with new changes, so might of miss something :p.  But a small question, this section seems to be reused four times, is it an option to put this section into the groovy file?

Reply:
Hm, maybe - a bit tricky as we need the `return` here but we could possibly make it something like:

`if (utils.dry_run()) return;` ?

Reply:
Makes sense.

Reply:
Good idea.
This PR looks good to me otherwise!